models/engine/file_storage.py

```diff
@@ -68,3 +68,19 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Retrieve one object"""
+        data = self.all(cls)
+        for k, v in data.items():
```

Comment:
Why you don't use the same  loop for this file? ` for k, v in data.items():`

Reply:
Ok Perfect, I forgot update this change into file_storage engine.